google-oauth-client-bom/pom.xml

```diff
@@ -107,12 +107,14 @@
       </plugin>
       <plugin>
         <artifactId>maven-javadoc-plugin</artifactId>
+        <version>3.2.0</version>
```

Comment:
where does the version come from now? The parent pom? We might not want to set it here.